lib/oracledb.js

```diff
@@ -87,6 +87,122 @@ function extend(oracledb) {
         value: 0,
         enumerable: true
       },
+      DBTYPE_VARCHAR: {
+        value: 1001,
```

Comment:
I can only get at some point, beyond that I don't have the knowledge, feel free to fix everything that needs to be corrected.